data/human/transport missions.txt

```diff
@@ -3582,3 +3582,743 @@ mission "Hauler VI cargo prototype 3"
+			`	Less than a minute later you hear Bunker on comms, "Drones are locked down and ready to go."`
+			`	The moment you pull out of the canyon pirate fighters appear on your scanners. You accelerate into space at full speed with the fighters in hot pursuit.`
+				launch
+			label blaster
+			`	You pull your blaster and order him to stand down. "Don't do this," he says. You're looking for the handcuffs when you notice a hum coming from outside. Suddenly you are raked with fire from one of the pirate escorts. You drop your blaster and try to lift off into space but another another escort appears, cutting you off and laying down a stream of fire. Your engines are hit and your ship falls from the sky and explodes on impact.`
+				explode
+
+	npc
+		personality heroic nemesis vindictive launching
+		government "Pirate"
+		ship "Quicksilver" "Don't Run"
+		ship "Quicksilver" "Obey"
+		ship "Quicksilver" "Work Is Freedom"
+		ship "Quicksilver" "Do As You're Told"
+		ship "Quicksilver" "Got You"
+		ship "Quicksilver" "Futile"
+
+	on visit
+		dialog `Your escort carrying the spy drones was left behind. Better take off and wait for them to enter the system.`
+
+
+mission "Journalism 5"
+	name "Slave trade"
+	description "Bring everyone back to <destination>."
+	minor
+	source "Maker"
+	destination "Moonshake"
+	landing
+	to offer
+		has "Journalism 4: done"
+	passengers 3
+	cargo "stealth drones" 1
+
+	on offer
+		conversation
+			branch experienced
+				"combat rating" > 100
+			`It was close but you make it back to Maker. Ale and River are excited to get on board and out of their uniforms and disguises. They are exhausted and eager to get home and process what they filmed, emotionally as well as literally. "That was horrible, but we got it done. Can we leave now?" asks Ale.`
+				launch
+			label experienced
+			`The pirates were no match for you. Ale and River are excited to get on board and out of their uniforms and disguises. They seem exhausted and eager to get home and process what they filmed, as well as emotionally. "That was awful. Can we leave now?" asks Ale.`
+				launch
+	on visit
+		dialog `You left the <cargo> behind. Better take off and wait for them to enter the system.`
+	on complete
+		payment 220000
+		log `Helped the Adelita collective gather video evidence of Syndicate participation in the slave trade.`
+		conversation
+			`They seem glad to be home. "Ugh. We saw such horrible things", says Ale, "and now we have to watch it over and over again to edit it into something presentable." She hands you a stack of credits. "There is a bonus in there for you. We really couldn't have done it without you. We will let you know when it's done. And we will definitely need you for some projects in the future. We'll be in touch."`
+
+
+mission "Journalism 6"
+	deadline 9
+	source "Trinket"
+	destination "Harmony"
+	to offer
+		has "Journalism 5: done"
+		or
+			has "FW Southern Recon 1B: done"
+			has "FW Southern Recon 1B (Turret): done"
```

Comment:
Im pretty sure if we do that we make the mission impossible to offer. Those missions are mutually exclusive. When you bring back the electron beam (either normal or turret, but not both like you put) the science team tells you Freya is waiting for you in a cafe. If more than one missing is offered at the same time they are ordered alphabetically. Since the F in "FW Southern Recon 1b" is before the J in "Journalism..." it should be offered right after you speak with Freya about going to get the Dreadnoughts.   Im pretty sure this worked well with i play tested. But i can check again.

Reply:
But as you mentioned before, this might be where its most cluttered with missions. It would be a bummer but i am not entirely opposed to removing most of the mission and just having an invitation to the opening of the expo. Another idea is instead of meeting up with them so they can go with you to the battle, you just meet up with one of their contacts in Zug who installs the cameras. Then after the battle you meet up with them for the expo. 
Again, i am happy to talk about this on discord as well but where? Im new to the platform. My username is joshu

Reply:
I do like your second option to avoid taking the mission out all together. Yeah, content creating is probably the best spot, but "content creating" encapsulates so many different things that story is rarely talked about. Besides, most people creating new stories right now are  experienced so detached from the rest of established space that there's rarely a need for them to talk about what they're writing. That does make it tough. My user's Wizrad; feel free to shoot me a DM or ping me in content-creating. I do partly like the idea of extending the Free Worlds breaks, but that might be a hard sell to a lot of the core members who would be the ones that decide whether that's something looking into. Also, if you don't know, make sure you load the hidden conversations on github. I had quite a few more suggested changes that you haven't addressed yet and wanted to make sure you'd seen they were there since you seem to have resolved the majority of changes that weren't automatically collapsed.

Reply:
Im not sure what you mean by "load the hidden conversations". And looks like i need your four digit tag to find you on discord.

Reply:
Number should be 3665. See attached image to see what I'm talking about. You want to click "Load more...".
![Screenshot (2)](https://user-images.githubusercontent.com/20505367/87447393-87298e00-c5c8-11ea-8999-7232704e2551.png)